bench/wtperf/config.c

```diff
@@ -134,9 +134,11 @@ config_free(CONFIG *cfg)
 	}
 
 	cleanup_truncate_config(cfg);
+	free(cfg->base_uri);
 	free(cfg->ckptthreads);
+	free((char *)cfg->partial_config);
 	free(cfg->popthreads);
-	free(cfg->base_uri);
+	free((char *)cfg->reopen_config);
 	free(cfg->workers);
```

Comment:
If you have to cast it, then I'd cast it to `(void *)`, because that's the type free takes.

But, this variable isn't really a const, we're casting it in other places so we can modify it:
```
        cfg->reopen_config = dcalloc(req_len, 1);
        if (cfg->readonly)
                snprintf((char *)cfg->reopen_config, req_len, "%s%s",
                    (char *)cfg->conn_config, READONLY_CONFIG);
        else
                snprintf((char *)cfg->reopen_config, req_len, "%s",
                    (char *)cfg->conn_config);
```
That cast of `cfg->conn_config` looks pretty odd, too.